balena/settings.py

```diff
@@ -31,14 +31,18 @@ class Settings(object):
     CONFIG_FILENAME = 'balena.cfg'
     DEFAULT_SETTING_KEYS = set(['pine_endpoint', 'api_endpoint', 'api_version',
                                 'data_directory', 'image_cache_time',
-                                'token_refresh_interval', 'cache_directory', 'timeout'])
+                                'token_refresh_interval', 'cache_directory', 'timeout',
+                                'image_maker_endpoint', 'image_maker_endpoint_version',
```

Comment:
Forgotten `image_maker_endpoint_version` occurrence
```suggestion
                                'image_maker_endpoint',
```